### ISSUE TITLE
Reusing same div to toggle overlay doesn't seem to work

### ISSUE THREAD

**Issue by peterlitvak** [openseadragon/openseadragon#950]
I have a div that is user as an overlay. When I call addOverlay the first time it shows overlay correctly. Then I call removeOverlay which makes overlay to disappear correctly. Then I call addOverlay again for the same div this time nothing is displayed.
It seems that removeOverlay sets the display: none for the element style

**Comment by boskar** [comment]
Related: https://github.com/openseadragon/openseadragon/issues/434

**Comment by iangilman** [comment]
@peterlitvak Thank you for the report! Would you be up for taking a look at making a possible fix? I suppose the trick would be doing the right thing for elements given to OSD as well as for elements OSD creates itself (since both are possible for overlays). 

As a workaround, does it work for you to unset the `display: none` on the element?

**Comment by peterlitvak** [comment]
Yes, when I need to show it again I just add $('#foo').show() after addOverlay('foo', ...) call. As far as the changes go will not be able to get to that for some time unfortunately, swamped with my dev work :)

**Comment by iangilman** [comment]
@peterlitvak Okay, good to know, thank you!

Hopefully someone else can dig into this.
